app/build.gradle

```diff
@@ -177,7 +198,8 @@ dependencies {
     focusCompile 'com.adjust.sdk:adjust-android:4.11.4'
     focusCompile 'com.google.android.gms:play-services-analytics:10.2.6' // Required by Adjust
 
-    geckoCompile(name: 'geckoview-latest', ext: 'aar')
+    // gecko_version = '60.0a1'
```

Comment:
What's up with this line? Do we need to set this in the future or can we remove this?